src/test/resources/Roman_Marcov/The first Feture file.feature

```diff
@@ -0,0 +1,22 @@
+Feature: The first automation tests
+
+
+  Scenario: Average of three numbers
+
+    Given Given three numbers "5", "4" and "6"
+    When Find average of three numbers
+    Then Average should be "5"
```

Comment:
add your initials RM because it will not work if someone will create the same steps